Automattic/woocommerce-payments
# Add WooPay on onboarding payment methods

#### Changes proposed in this Pull Request









Add WooPay icon on the onboarding payment methods list and a link to its ToS when the merchant country is available to use WooPay.

#### Testing instructions





* On WCPay DevTools, enable `Force onboarding`, click `Submit` and then `Reonboard`.
* Set your store country is `US`, click the `Payments` link on the sidebar.
* It should show the WooPay icon on the payment methods list and a link to the WooPay terms of service, the design should match the one on 2020-gh-Automattic/woopay Figma.

![Screenshot 2023-07-31 at 6 54 17 PM](https://github.com/Automattic/woocommerce-payments/assets/1693223/3a7244ad-7965-40f3-99f1-41be1da774ca)

* Change your store country.
* Click the `Payments` link on the sidebar.
* It should not show the WooPay icon on the payment methods list and a link to the WooPay terms of service.

The list of available countries is updated everyday on #6562.

-------------------


**Post merge**
